index.bs

```diff
@@ -4226,22 +4227,23 @@ keyword is used, this includes <code class="idl">entries</code>,
 For read–write setlikes, it also includes <code class="idl">add</code>,
 <code class="idl">clear</code>, and <code class="idl">delete</code> methods.
 
-Setlike interfaces must not
-have any [=interface members=]
-named "<code>entries</code>", "<code>forEach</code>",
-"<code>has</code>", "<code>keys</code>",
-"<code>size</code>", or "<code>values</code>",
-or have any [=inherited interfaces=]
-that have [=members=] with these names.
-Read–write setlike interfaces must not
-have any [=attributes=]
-or [=constants=] named
-"<code>add</code>", "<code>clear</code>",
-or "<code>delete</code>", or have any [=inherited interfaces=]
-that have [=attributes=] or [=constants=] with these names.
-
-Note: Operations named "<code>add</code>", "<code>clear</code>",
-or "<code>delete</code>" are allowed on read–write setlike
+Setlike interfaces and their [=inherited interfaces=] must not have any [=regular operations=],
+[=regular attributes=] or [=constants=] named
```

Comment:
This forbids
```webidl
interface A { void entries(); };
interface B : A { readonly setlike<long>; };
```
but not
```webidl
interface A { readonly setlike<long>; };
interface B : A { void entries(); };
```

Should it?

Reply:
That is a really good question....  Seems like disallowing it until there's a use case that might want it, and then deciding how it should work, is reasonable...

Reply:
Done.